blog-posts/getting-started-gatsby-next-remix.md

```diff
@@ -0,0 +1,306 @@
+
+- `path`: This key determines which path will be pre-rendered. In the in-code comment below the structure of `path` is given as an example.
+- `fallback`: You can set the value of fallback to `true`, `false` or `blocking`
+    - `false` → Any paths not returned by `getStaticPaths` will end up in a 404 page
+    - `true` → On the first request to a not generated path, Next.js will serve a fallback version of the page, instead of a 404 page. The HTML and JSON of this path is now generated in the background and and then the JSON is sent to the requesting browser. Now the fallback page changes into the full page. Next.js will add this path to the pre-rendered pages in order to be able to provide the already generated page for further requests.
+      This is useful if you have a page with a lot of static pages because then not all pages have to be generated at built time, which makes the built much faster.
+    - `blocking` → Same procedure as for `fallback: true`, except that there is no fallback while the HTML is being generated. There are only very special uses cases in which this makes sense, such as AMP.
+
+Remember to name the file after the relevant key from the `params` object with square brackets, in our case `[path].tsx`.
+To load the data for a blogpost the function `getStaticProps` is needed. This function now receives `{params}`, which can be used to determine the path and the depending data. Finally, in the actual page component, the data can be accessed via props (here via `{postData}`).
+
+```tsx
+export async function getStaticPaths() {
+    //     paths: [
+    //         {params: {path: 'first-blogpost'}},
+    //         {params: {path: 'second-blogpost'}},
+    //         ...
+    //         {params: {path: 'fifth-blogpost'}}
+    //     ],
+    const paths = getAllPostPaths()
+
+    return {
+        paths,
+        fallback: false
+    }
+
+export async function getStaticProps({params}: { params: PostsData }) {
+    const postData = await getPostData(params.path)
+    return {
+        props: {
+            postData
+        }
+    }
+}
+
+export default function Post({postData}: { postData: PostData }) {
+    return (
+            <div dangerouslySetInnerHTML={{__html: postData.contentHtml}}/>
+    )
+}
+```
+
+
+> 💡 The object returned by `getStaticProps` can include other keys besides `props`, such as `revalidate`. `revalidate` is set to `false` by default, but can also contain the amount in seconds after which a page re-generation can occur. In this way, individual static pages can be changed at a certain interval after being built. This is called [Incremental Static Regeneration (ISR)](https://vercel.com/docs/concepts/next.js/incremental-static-regeneration).
+
+
+#### **Remix**
+
+Remix supports only server side rendering. An example of this can be found in the section below.
```

Comment:
Remix explicitly only does SSR, because they have an architecture in mind where SSR is superior or equal to SSG in all manners.

Some of their reasons can be found in their latest article about "Remix vs Next". 
https://remix.run/blog/remix-vs-next

tl;dr:
- CDN with proper caching headers will always be fast, except for cold requests (like the first request)
- If the backend is so slow that cold requests are a problem, the backend should be improved or cached
- If pages are visited so rarely that they are always cold, it's a business problem and not a technical

Reply:
We should add this to this part. Otherwise it looks like it's a problem or big downside.

Reply:
This was fixed. Thanks for the text.